### ISSUE TITLE
TheLink connection issues 

### ISSUE THREAD

**Issue by joehand** [karissa/hyperdrive-archive-swarm#21]
We've traced down TheLink's connection issue to an issue with hyperdrive-archive-swarm. I used the discovery-swarm dependency directly and that fixed it. 

This may have been an issue from going from 3.x.x to 4.0, but downgrading didn't work initially.

**Comment by okdistribute** [comment]
what's the working code you are using for discovery-swarm? Let's look at the difference between it and archive swarm to see where the bug might be.

**Comment by joehand** [comment]
[This swarm usage](https://github.com/joehand/dat-js/blob/TheLinkNet/index.js#L264) worked. I'm not sure if its one of the options or something else throwing it off, Mathias and I tried to debug a bit but couldn't find anything different.

We talked today about removing the webrtc support for now from dat & dat-js, and moving to using discovery-swarm directly. We want to make sure to get these network issues solved. Having these external modules is making it harder; On TheLink's issue the dat doctor & dat cli had different network results.
